src/main/java/application/AbstractBaseController.java

```diff
@@ -373,6 +373,12 @@ public ExceptionResponseBean handleLockError(FormplayerHttpRequest req, Exceptio
         return new ExceptionResponseBean("User lock timed out", req.getRequestURL().toString());
     }
 
+    @ExceptionHandler({InterruptedRuntimeException.class})
+    @ResponseBody
+    public ExceptionResponseBean handleInterruptException(FormplayerHttpRequest req, Exception exception) {
+        return new ExceptionResponseBean("Request interrupted, please try again.", req.getRequestURL().toString());
```

Comment:
Is this going to show up to a user? If so, this isn't an OK message, it won't mean anything to them.

The user facing message should avoid using words that they won't understand in favor of something like

"An issue prevented us from processing your previous action, please try again"